app.py

```diff
@@ -1658,12 +1667,23 @@ def client_messages():
     d.update(YamlFile.for_file(
         f'content/client-messages/{g.lang}.yaml').to_dict())
 
+    # A/B testing
+    # If alternative error messages are available for this user
+    # use alternative file for translation
+    if alternative_error_messages.is_available():
+        d.update(YamlFile.for_file(f'content/client-messages/{g.lang}_test.yaml').to_dict())
+
+    # If alternative error message is set use alternative .yaml file
+
     response = make_response(render_template(
         "client_messages.js", error_messages=json.dumps(d)))
 
-    if not is_debug_mode():
+    if is_debug_mode():
+        response.cache_control.max_age = 0
+    else:
         # Cache for longer when not developing
         response.cache_control.max_age = 60 * 60  # Seconds
+
```

Comment:
I think @rix0rrr is also working on some re-written of the `client-messages` structure. Might have to check with him if this is still suitable with the new approach.

Reply:
Good catch, thank @TiBiBa! It would be most helpful to me if we can put the new error messages in the same file, but give them different keys.

So, instead of:

```
client-messages/nl.yaml
   Transpile_error: "We konden je code niet goed lezen."

client-messages/nl.yaml
   Transpile_error: "Dat ging fout. Probeer het opnieuw 🙂"
```

I'd rather see:

```
client-messages/nl.yaml
   Transpile_error: "We konden je code niet goed lezen."
   NEW_Transpile_error: "Dat ging fout. Probeer het opnieuw 🙂"
```

Is there a reason not to do it that way?

(Is the reason Weblate?)